### DISCUSSION TITLE
containsRect on Component

### DISCUSSION THREAD

**Discussion by oloshe** [flame-engine/flame#1603] [Q&A] [answered]
Is there a simple way to judge whether two rectangles intersect or contain? such like `component.containsRect`.
I know `CollisionCallbacks` can determine whether two rectangles intersect, but not whether they contain?

**Comment by oloshe** [comment, marked as answer]
It seems that I found it finally, use `aabb.containsAabb2()` can solve it.

**Reply by spydon** [reply]
You can also do `toRect` on `PositionComponent`s and then compare the `Rect`s with `rect.intersect(otherRect)`.
